doc/userguide/configuration/suricata-yaml.rst

```diff
@@ -1065,21 +1065,20 @@ flow-engine goes into the emergency-mode. In this mode, the engine
 will make use of shorter time-outs. It lets flows expire in a more
 aggressive manner so there will be more space for new Flows.
 
-There are two options: emergency_recovery and prune_flows. The
-emergency recovery is set on 30. This is the percentage of prealloc'd
-flows after which the flow-engine will be back to normal (when 30
-percent of the 10000 flows is completed).
+Emergency_recovery defines the percentage of flows that the engine needs to
```

Comment:
nit: if this is referring to the option below, perhaps we should let it be exact so all small letters